<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkAnalyzerChannelMenu</class>
 <widget class="QWidget" name="NetworkAnalyzerChannelMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>286</width>
        <height>693</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="scopy::gui::MenuHeader" name="widgetMenuHeader" native="true"/>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetReferenceSubsSep" native="true"/>
       </item>
       <item>
        <widget class="scopy::gui::CustomSwitch" name="btnRefChn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
min-height: 30px;
max-height: 30px;
min-width: 280px;
max-width: 280px;
background-color: black;
border-radius: 4px;
}

QPushButton:disabled {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.501, y2:0, stop:0 #727273, stop:1 #141416);
}

QWidget#handle {
min-height: 30px;
max-height: 30px;
min-width: 142px;
max-width: 142px;
background-color: #4a64ff;
border-radius: 2px;
}

QWidget#handle:disabled {
background-color: #aaaaaa;
}

QLabel {
margin-top: 6px;
background-color: transparent;
line-height: 19px;
}

QLabel#on {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:140px;
max-width:140px;
qproperty-alignment: AlignCenter AlignCenter;
margin-left: 0px;
color: white;
}

QLabel#on:disabled {
color: rgba(255,255,255,102);
}

QLabel#off {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:140px;
max-width:140px;
margin-left: 140px;
color: white;
qproperty-alignment: AlignCenter AlignCenter;
}

QLabel#off:disabled {
color: rgba(255,255,255,51);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="leftText" stdset="0">
          <string>Channel 1</string>
         </property>
         <property name="rightText" stdset="0">
          <string>Channel 2</string>
         </property>
         <property name="duration_ms" stdset="0">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>15</number>
         </property>
         <item row="4" column="1">
          <layout class="QVBoxLayout" name="vLayoutCaptureDelay">
           <property name="topMargin">
            <number>10</number>
           </property>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="vLayoutOffset"/>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="vLayoutPushDelay">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="vLayoutAmplitude"/>
         </item>
         <item row="3" column="1">
          <layout class="QVBoxLayout" name="vLayout_19">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_19">
             <property name="text">
              <string>Gain Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbBoxResponseGain">
             <item>
              <property name="text">
               <string>Automatic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Low</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>High</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="vLayout_22">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_12">
             <property name="text">
              <string>DC Filtering</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="scopy::gui::CustomSwitch" name="btnDcFilter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="leftText" stdset="0">
              <string>On</string>
             </property>
             <property name="rightText" stdset="0">
              <string>Off</string>
             </property>
             <property name="duration_ms" stdset="0">
              <number>0</number>
             </property>
             <property name="polarity" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="scopy::gui::SubsectionSeparator" name="widgetResponseSubsSep" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>15</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetSweepSubsSep" native="true"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayoutSweep">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>15</number>
         </property>
         <item row="6" column="1">
          <layout class="QVBoxLayout" name="vLayoutSamplesCount">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item row="6" column="0">
          <spacer name="vSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>12</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="scopy::gui::CustomSwitch" name="btnIsLog">
           <property name="styleSheet">
            <string notr="true">QPushButton {
min-height: 30px;
max-height: 30px;
min-width: 280px;
max-width: 280px;
background-color: black;
border-radius: 4px;
}

QPushButton:disabled {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.501, y2:0, stop:0 #727273, stop:1 #141416);
}

QWidget#handle {
min-height: 30px;
max-height: 30px;
min-width: 142px;
max-width: 142px;
background-color: #4a64ff;
border-radius: 2px;
}

QWidget#handle:disabled {
background-color: #aaaaaa;
}

QLabel {
margin-top: 6px;
background-color: transparent;
line-height: 19px;
}

QLabel#on {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:140px;
max-width:140px;
qproperty-alignment: AlignCenter AlignCenter;
margin-left: 0px;
color: white;
}

QLabel#on:disabled {
color: rgba(255,255,255,102);
}

QLabel#off {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:140px;
max-width:140px;
margin-left: 140px;
color: white;
qproperty-alignment: AlignCenter AlignCenter;
}

QLabel#off:disabled {
color: rgba(255,255,255,51);
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="leftText" stdset="0">
            <string>Logarithmic</string>
           </property>
           <property name="rightText" stdset="0">
            <string>Linear</string>
           </property>
           <property name="duration_ms" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="sweepRangeLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayoutSweepAverage">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>3</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinBoxAveraging">
             <property name="styleSheet">
              <string notr="true">QSpinBox{
font-size: 14px;
height:24px;
}


QSpinBox[invalid=true] {
border-color: red;
}
QSpinBox[valid=true] {
border-color: green;
}</string>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblAveraging">
             <property name="text">
              <string>Average</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnApplyAverage">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border-width: 0px;
border-radius: 3px;
min-width: 64px;
}

/*
QPushButton:enabled:!pressed { background-color: #4a64ff; }
QPushButton[valid=true]:enabled:!pressed { background-color: #4a64ff; }
QPushButton[invalid=true]:enabled:!pressed { background-color: #4a64ff; }
*/
</string>
             </property>
             <property name="text">
              <string>Apply</string>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayoutSweepPeriod">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>3</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lblPeriods">
             <property name="text">
              <string>Periods</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinBoxPeriods">
             <property name="styleSheet">
              <string notr="true">QSpinBox{
font-size: 14px;
height:24px;
}
QSpinBox[invalid=true] {
border-color: red;
}
QSpinBox[valid=true] {
border-color: green;
}</string>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnApplyPeriod">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border-width: 0px;
border-radius: 3px;
min-width: 64px;
}

/*
QPushButton:enabled:!pressed { background-color: #4a64ff; }
QPushButton[valid=true]:enabled:!pressed { background-color: #4a64ff; }
QPushButton[invalid=true]:enabled:!pressed { background-color: #4a64ff; }
*/</string>
             </property>
             <property name="text">
              <string>Apply</string>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetDisplaySubsSep" native="true"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_9" rowminimumheight="0,0,0,0,0,0,0">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>15</number>
         </property>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="vLayoutPhaseMin"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbl_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="vLayoutPhaseMax"/>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="vLayoutMagMin"/>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="vLayoutMagMax"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delta Label</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="scopy::gui::CustomSwitch" name="btnDelta">
           <property name="text">
            <string/>
           </property>
           <property name="leftText" stdset="0">
            <string>On</string>
           </property>
           <property name="rightText" stdset="0">
            <string>Off</string>
           </property>
           <property name="duration_ms" stdset="0">
            <number>0</number>
           </property>
           <property name="polarity" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetBufferPreviewerSubsSep" native="true"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vLayout_21">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="hLayout_20">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="scopy::gui::CustomSwitch" name="btnBufferPreview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="hSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnViewInOsc">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;
}</string>
             </property>
             <property name="text">
              <string>ViewInOsc</string>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayout_23">
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnPrev">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;

}</string>
             </property>
             <property name="text">
              <string>Previous</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/common/ic arrow left.svg</normaloff>:/icons/common/ic arrow left.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>12</width>
               <height>12</height>
              </size>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="hSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNext">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;

text-align: right;
padding-left: 20px;
}</string>
             </property>
             <property name="text">
              <string>     Next</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/common/ic arrow right.svg</normaloff>:/icons/common/ic arrow right.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>12</width>
               <height>12</height>
              </size>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="hSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::CustomSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/custom_switch.hpp</header>
  </customwidget>
  <customwidget>
   <class>scopy::gui::SubsectionSeparator</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/subsection_separator.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scopy::gui::MenuHeader</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/menu_header.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
